include/ccf/service/node_info_network.h

```diff
@@ -99,6 +99,9 @@ namespace ccf
       /// this interface. std::nullopt means everything is accepted.
       std::optional<std::vector<std::string>> accepted_endpoints = std::nullopt;
 
+      /// Timeout for forwarded RPC calls (in milliseconds)
+      std::optional<size_t> forwarding_timeout = std::nullopt;
```

Comment:
```suggestion
      std::optional<size_t> forwarding_timeout_ms = std::nullopt;
```

We try to include units in names, wherever ambiguity might exist.